domain/lib/src/model/share/mailing_list_id.dart

```diff
@@ -38,7 +38,8 @@ import 'package:equatable/equatable.dart';
 
 class MailingListId with EquatableMixin {
   String _uuid;
-    String get uuid => _uuid;
+
```

Comment:
we are using this way for defining `public get method` of a `private field`. Please remove extra line between
```
String _uuid;
String get uuid => _uuid;

Reply:
I've updated. Plz help me review.